examples/server-communication__visit-2nd-domain/cypress/integration/spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+describe('Two domains', () => {
+  it('visits 1nd domain', () => {
+    cy.visit('https://www.cypress.io/')
+    // there are several GitHub links on the page, make sure
+    // to use the selector that returns a single item
+    cy.get('header [aria-label="Check out our github page"]')
+    .should('have.length', 1)
+    // from the jQuery wrapping <a href="https://github.io ..." />
+    // get the "href" value
+    .invoke('attr', 'href')
+    .then((url) => {
+      // save the value in the plugins process
+      // that is preserved between test and browser restarts
+      expect(url).to.be.a('string')
+      cy.task('saveUrl', url)
+    })
+  })
+
+  it('visits 2nd domain', () => {
+    // we assume the first test has finished
+    // and stored the url
+    cy.task('getUrl')
+    .then((url) => {
+      expect(url).to.be.a('string')
+      cy.visit(url)
```

Comment:
I don't really like the idea of this visiting a GitHub domain - if GitHub is down, our tests will fail which blocks deployment from our example recipes. Although I'm not sure what cross domain we could visit here to show this that's under our control - actually maybe it could be a `https` versus `http` domain? Those are actually not considered single-domain.

Reply:
but github is such a beautiful example, if it starts going down ever breaking our recipes, I will change it